graph-manager-docs/source/managed-federation/setup.mdx

```diff
@@ -0,0 +1,134 @@
+        name: "Products"
+        url: "http://products-graphql.svc.cluster.local:4001/"
+        endpoint: {
+          url: "http://localhost:4000"
+        }
+      }
+    };
+    ```
+
+    * The `name` field uniquely identifies the service within your data graph. Its value appears throughout the Graph Manager UI.
+    * The `url` field points to your service's GraphQL endpoint. To secure your data graph, this should be a URL that can be reached by your [gateway](https://www.apollographql.com/docs/apollo-server/federation/gateway/), but _cannot_ be reached via the public internet.
+
+3. From your project's root directory, run the following:
+
+    ```
+    apollo service:push --tag=staging
+    ```
+
+    The CLI obtains all necessary configuration from your project's `.env` and `apollo.config.js` files, then pushes your schema to Graph Manager under the `staging` variant.
+
+> When you're ready to register your services in your production environment, you can either remove the `--tag` option to push to the default variant (named `current`) or replace `staging` with a new variant (such as `production`).
+
+## 2. Configure continuous delivery
+
+With managed federation, Graph Manager (_not_ your gateway) is responsible for combining your individual schemas into a single federated schema. Consequently, **each service in your data graph must re-register its schema every time the schema changes.** Otherwise, your gateway can't obtain the latest federated schema from Graph Manager.
+
+The best way to accomplish this is to add the `apollo service:push` call to your continuous delivery pipeline. Every time you push a schema change to production, this call should automatically accompany that push.
+
+For more information on best practices for updating your services, see [Pushing configuration updates safely](./advanced-topics/#pushing-configuration-updates-safely).
+
+## 3. Modify the gateway
+
+Because your gateway isn't responsible for creating your federated schema anymore, you need to modify its initialization.
+
+> This section assumes you are using Apollo Server with the `@apollo/gateway` library as your gateway.
+
+### Remove the `serviceList` argument
+
+A bare-bones initialization of an `ApolloGateway` object looks like this:
+
+```js
+const gateway = new ApolloGateway({
+  serviceList: [
+    { name: 'accounts', url: 'http://localhost:4001' },
+    // Additional services defined here
+  ],
+});
+```
+
+The `serviceList` argument specifies the name and URL for each of your implementing services. With managed federation, your gateway obtains this information from Graph Manager instead.
+
+**Remove the `serviceList` argument from your `ApolloGateway` constructor entirely.**
+
+### Connect the gateway to Graph Manager
+
+Like your implementing services, your gateway uses an API key to identify itself to Graph Manager. Create an API key from your graph's Settings page in the [Graph Manager UI](https://engine.apollographql.com/).
```

Comment:
I think it's worth a link / note to the fact that it's not _actually_ querying graph manager, but using indirection for stability